resources/keywords/login.resource

```diff
@@ -24,7 +26,21 @@ Login To Openshift
     Input Password    ${txtBxPassword}    ${OCP_ADMIN_USER.PASSWORD}
     Click Element    ${btnLogin}
     Page Should Not Contain Element    ${lblLoginError}    message=Login Failed
-    Wait Until Keyword Succeeds    10x    2s    Page Should Contain Element    ${titleOverview}
+    Wait Until Keyword Succeeds    6x    5s    Wait Until Page Contains Element    ${btnPerspective}
+    User Selects Administrator View
+
+User Selects ${perspective} View
```

Comment:
In simple python it can be done in 3-4 lines :-)

Reply:
This script will change the perspective of the Openshift application to given value i.e Administrator and Developer. 
The script created in generic way as it can be used anywhere, in some part of execution like first time login -  the application DOM takes too much time to be ready and some parts it loads with the screen where we left. The script is capable of handling these situation and that's the reason it was being too lengthy. But IMHO, the logic what is been scripted will also need on python file. So, It would be somewhat similar on python.